Method does not override or implement a method from a supertype

Hello!

With React Native v0.47.1 I have the following error while running `react-native run-android`:

```
/node_modules/react-native-immersive/android/src/main/java/com/rnimmersive/RNImmersivePackage.java:20: error: method does not override or implement a method from a supertype
  @Override
  ^
1 error
```

I can bypass this error only after removing `@Override` from:

```java
 @Override
  public List<Class<? extends JavaScriptModule>> createJSModules() {
    return Collections.emptyList();
  }
```
This is the similar issue as [this](https://github.com/oblador/react-native-vector-icons/issues/515)

--- a/android/src/main/java/com/rnimmersive/RNImmersivePackage.java
+++ b/android/src/main/java/com/rnimmersive/RNImmersivePackage.java
@@ -17,7 +17,6 @@ public class RNImmersivePackage implements ReactPackage {
     return modules;
   }
 
-  @Override
   public List<Class<? extends JavaScriptModule>> createJSModules() {
     return Collections.emptyList();
   }

